patroni/version.py

```diff
@@ -1 +1 @@
-__version__ = '2.1.3'
+__version__ = '2.2.0'
```

Comment:
Can we use something like `2.1.3-static-primary` as a version string. It will show the version this has branched off, together with indicating this is not a mainline release, but a fork.

Reply:
Sure, however due to packaging issues, I am not even including this versioning info in the container image, as without going through a full packaging process, the version will cause conflicts with package resolution. So at this point, the version just reflects the ultimate intent of upstreaming this changeset. Thoughts?